### ISSUE TITLE
Author still doesn't believe us

### ISSUE THREAD

**Issue by xorbotx** [0x48piraj/fadblock#212]
Based on his comments at the chromewebstore. But I can understand, these issues seem to be really hard to diagnose when they happen sporadically for some people and not others, and can't easily be reproduced. I stopped using Fadblock on opera a week or so ago after getting the adblocker warning on every ad for a couple days straight. I tested by disabling fadblock and the video played normally, then enabled fadblock and the adblocker warning came up, etc. But yesterday I tried enabling it again and every ad has been skipped so far, so it's working again for me.

EDIT: it's getting detected again two days later

**Comment by richpav** [comment]
Stopped working for me with Thorium somewhere between Jul 18-19. Maybe location is relevant? I'm in Japan.

**Comment by 0x48piraj** [comment]
As said, the extension works and is daily used by me and most of my peers. The moment it really stops working, I fix it up.

To all those who are running other adblockers side by side, I cannot help, they are getting detected. Worse yet, YouTube puts a tracker when detected and that causes the "I deleted the other adblockers and I am still getting detected" queries.

Even after logging out, clearing out the cookies and logging back in sometimes doesn't fix the issue for some (it fixed for most of us) hinting they are marking the accounts using adblockers server-side but I can't say for sure as FadBlock works perfectly for me as of now and many of the supporters I have talked to over the email.

Move to https://github.com/0x48piraj/fadblock/issues/196
